cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
```

Comment:
In my opinion, we should not have any package blocks unit testing, maybe with the exception of 'utils' kind of package.
Having `InstallType` as a function will block unit testing covering the usage.